src/main/resources/blockchain/default-genesis.json

```diff
@@ -0,0 +1,8905 @@
+    "1518627b88351fede796d3f3083364fbd4887b0c": {"balance": "16000000000000000000000"},
+    "15224ad1c0face46f9f556e4774a3025ad06bd52": {"balance": "13370000000000000000"},
+    "152f2bd229ddf3cb0fdaf455c183209c0e1e39a2": {"balance": "2000000000000000000000"},
+    "152f4e860ef3ee806a502777a1b8dbc91a907668": {"balance": "600000000000000000000"},
+    "153c08aa8b96a611ef63c0253e2a4334829e579d": {"balance": "394000000000000000000"},
+    "153cf2842cb9de876c276fa64767d1a8ecf573bb": {"balance": "2000000000000000000000"},
+    "153ef58a1e2e7a3eb6b459a80ab2a547c94182a2": {"balance": "96000000000000000000000"},
+    "154459fa2f21318e3434449789d826cdc1570ce5": {"balance": "2000000000000000000000"},
+    "1547b9bf7ad66274f3413827231ba405ee8c88c1": {"balance": "17300000000000000000000"},
+    "1548b770a5118ede87dba2f690337f616de683ab": {"balance": "527558000000000000000"},
+    "15528350e0d9670a2ea27f7b4a33b9c0f9621d21": {"balance": "4000086000000000000000"},
+    "155b3779bb6d56342e2fda817b5b2d81c7f41327": {"balance": "50200000000000000000"},
+    "1565af837ef3b0bd4e2b23568d5023cd34b16498": {"balance": "393284000000000000000"},
+    "15669180dee29598869b08a721c7d24c4c0ee63f": {"balance": "1000000000000000000000"},
+    "1572cdfab72a01ce968e78f5b5448da29853fbdd": {"balance": "5061500000000000000000"},
+    "157559adc55764cc6df79323092534e3d6645a66": {"balance": "6000000000000000000000"},
+    "1578bdbc371b4d243845330556fff2d5ef4dff67": {"balance": "100000000000000000000"},
+    "157eb3d3113bd3b597714d3a954edd018982a5cb": {"balance": "2000000000000000000000"},
+    "1584a2c066b7a455dbd6ae2807a7334e83c35fa5": {"balance": "130000000000000000000"},
+    "15874686b6733d10d703c9f9bec6c52eb8628d67": {"balance": "2000000000000000000000"},
+    "158a0d619253bf4432b5cd02c7b862f7c2b75636": {"balance": "135733000000000000000"},
+    "1598127982f2f8ad3b6b8fc3cf27bf617801ba2b": {"balance": "173000000000000000000"},
+    "159adce27aa10b47236429a34a5ac42cad5b6416": {"balance": "31867951000000000000000"},
+    "15a0aec37ff9ff3d5409f2a4f0c1212aaccb0296": {"balance": "1000000000000000000000"},
+    "15aa530dc36958b4edb38eee6dd9e3c77d4c9145": {"balance": "2000000000000000000000"},
+    "15acb61568ec4af7ea2819386181b116a6c5ee70": {"balance": "31000000000000000000000"},
+    "15b96f30c23b8664e7490651066b00c4391fbf84": {"balance": "410650000000000000000"},
+    "15c7edb8118ee27b342285eb5926b47a855bc7a5": {"balance": "20000000000000000000"},
+    "15d99468507aa0413fb60dca2adc7f569cb36b54": {"balance": "2000000000000000000000"},
+    "15dbb48c98309764f99ced3692dcca35ee306bac": {"balance": "150000000000000000000000"},
+    "15dcafcc2bace7b55b54c01a1c514626bf61ebd8": {"balance": "9400000000000000000000"},
+    "15e3b584056b62c973cf5eb096f1733e54c15c91": {"balance": "936702000000000000000"},
+    "15ebd1c7cad2aff19275c657c4d808d010efa0f5": {"balance": "200550000000000000000"},
+    "15ee0fc63ebf1b1fc49d7bb38f8863823a2e17d2": {"balance": "191000000
```

Comment:
That's good point. I don't know. This genesis block is straight from geth classic